docs/static/monitoring-apis.asciidoc

```diff
@@ -114,28 +224,27 @@ The node stats API retrieves runtime stats about Logstash.
 GET /_node/stats/<types>
 --------------------------------------------------
 
-////
-COMMENTED OUT until Logstash supports multiple pipelines: To retrieve all stats per pipeline, use the `_pipelines/stats` endpoint:
-
-[source,js]
---------------------------------------------------
-GET /_pipelines/stats/<types>
---------------------------------------------------
-////
-
 Where `<types>` is optional and specifies the types of stats you want to return.
 
 By default, all stats are returned. You can limit this by combining any of the following types: 
 
 [horizontal]
 `events`::
-Gets event information since startup. 
+Gets event information since startup.
 `jvm`::
-Gets JVM stats, including stats about threads. coming[5.0.0-alpha3,Adds thread count]
+Gets JVM stats, including stats about threads. added[5.0.0-alpha3,Adds thread count]
 `process`::
-Gets process stats, including stats about file descriptors, memory consumption, and CPU usage. coming[5.0.0-alpha3]   
+Gets process stats, including stats about file descriptors, memory consumption, and CPU usage. added[5.0.0-alpha3] 
+`mem`::
+Gets memory usage stats. coming[5.0.0-alpha4] 
+`pipeline`::
+Gets stats for a pipeline, including stats about each configured input, filter, or output stage. coming[5.0.0-alpha4,Stats for input stages are not yet available] 
+
+//REVIEWERS: In the latest Logstash build available on June 24, the APIs did not behave exactly as they are documented in https://github.com/elastic/logstash/issues/5475. /_node/stats/pipeline returns the expected results, but I couldn't specify anything below that (such as /_node/stats/pipeline/events). So I've documented the end points accordingly. I also noticed that mem appears in the returned JSON doc. Can you help me by suggesting a better/more complete description>? 
```

Comment:
This is correct. you cannot do `/_node/stats/pipeline/events` now. Only supported option is `/_node/stats/pipeline`